<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aufgabe2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="sourceImage1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sourceImage2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="destinationImage">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="blendSlider">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>29</y>
      <width>20</width>
      <height>321</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAufgaben">
    <property name="title">
     <string>Aufgaben</string>
    </property>
    <addaction name="action_ffnen"/>
   </widget>
   <addaction name="menuAufgaben"/>
  </widget>
  <action name="action_ffnen">
   <property name="text">
    <string>Ã–ffnen ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
